client/src/components/Input/Input.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FormEvent, FC } from 'react';
+
+interface InputI {
+  onChange: (e: FormEvent<HTMLInputElement>) => void;
+  value: string;
+  className: string;
+  type: string;
+  name: string;
+  id: string;
+  placeholder: string;
+}
+export const Input: FC<InputI> = ({
+  onChange,
+  value,
+  className,
+  type,
+  name,
+  id,
+  placeholder,
+}) => {
+  return (
```

Comment:
reduce return

Reply:
End with code correction but I`m not changed anything in map and order logic